doc/source/ray-core/handling-dependencies.rst

```diff
@@ -557,3 +557,34 @@ Here is a list of different use cases and corresponding URLs:
 Once you have specified the URL in your ``runtime_env`` dictionary, you can pass the dictionary
 into a ``ray.init()`` or ``.options()`` call. Congratulations! You have now hosted a ``runtime_env`` dependency
 remotely on GitHub!
+
+
+Debugging
+---------
+If runtime envs cannot be set up (e.g., network issues, download failures, etc.), Ray will fail to schedule tasks/actors 
```

Comment:
Nit, would prefer to use either "runtime environment" or "`runtime_env`" everywhere rather than "runtime env".  It's minor, I might not have been adhering to this before